src/btree/bt_vrfy.c

```diff
@@ -240,14 +240,18 @@ __wt_verify(WT_SESSION_IMPL *session, const char *cfg[])
         goto done;
 
     /*
-     * Get a list of the checkpoints for this file. Empty objects have no checkpoints, in which case
-     * there's no work to do.
+     * Get a list of the checkpoints for this file. Empty objects and ingest tables have no
+     * checkpoints, in which case there's no work to do.
      */
     WT_ERR_NOTFOUND_OK(__wt_meta_ckptlist_get(session, name, false, &ckptbase, NULL), true);
     if (ret == WT_NOTFOUND) {
         ret = 0;
         goto done;
-    }
+    } else if (WT_SUFFIX_MATCH(name, ".wt_ingest"))
+        WT_ERR_MSG(session, WT_ERROR,
+          "verify (layered): ingest table %s unexpectedly has checkpoints. "
+          "This is a fatal violation as the ingest table does not get checkpointed.",
```

Comment:
I get a different `s_all` formatting on my platform:

```suggestion
          "verify (layered): ingest table %s unexpectedly has checkpoints. This is a fatal "
          "violation as the ingest table does not get checkpointed.",
```

Reply:
okay now thats weird. I would expect `s_all` to result in the same formatting...